src/main/java/com/wepay/model/data/HeaderData.java

```diff
@@ -4,4 +4,7 @@
     public String accessToken;
     public String riskToken;
     public String clientIP;
+
```

Comment:
Allows to set specific timeouts per request if needed

Reply:
We have 60s timeout from server side but received several requests from contributors to make client-side timeout configurable. Our team is still having discussion internally and will modify it after we achieving a conclusion. So let's hold off making client-side timeout configurable currently.